source/manuals/alerting-with-prometheus-for-paas.html.md.erb

```diff
@@ -40,7 +40,7 @@ An alerting rule file will look something like this:
   groups:
 - name: Your team name
   rules:
-  - alert: RequestsExcess5xx
+  - alert: TeamName_RequestsExcess5xx
```

Comment:
this is contingent on our decision in alphagov/prometheus-aws-configuration-beta#82

Reply:
(but for the moment let's go with it as written, we can update later)